dictionary/README.md

```diff
@@ -2,4 +2,4 @@
 Please write a couple of sentences explaining your choice of restart
 strategy.
 
-
+I chose the permanent strategy. If the dictionary Agent isn't restarted, no new games can be created.  However, an argument could be presented in limiting the amount of times it should restart in a certain time so it doesn't slow down the system. 
```

Comment:
# +2

I believe this is the correct answer, but you're only the second person to give it. I'm so happy I can't stop myself adding 2 points.